insanely_fast_whisper_rocm/cli/commands.py

```
@@ -207,12 +448,21 @@ def _run_task(*, task: str, audio_file: Path, **kwargs) -> None:  # noqa: C901
+        if task == "transcribe":
+            error_label = "Transcription"
+        elif task == "translate":
+            error_label = "Translation"
+        else:
+            error_label = task.capitalize()
+        click.secho(f"\n❌ {error_label} error: {message}", fg="red", err=True)
         if debug:
             logger.exception("%s error details", task)
         sys.exit(1)
 
     except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.7)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)